tools/mo/openvino/__init__.py

```diff
@@ -51,6 +51,7 @@
     from openvino._pyopenvino import RemoteContext
     from openvino._pyopenvino import RemoteTensor
     from openvino._pyopenvino import Op
+    from openvino._pyopenvino import OpExtension
```

Comment:
```suggestion
```

please don't update legacy files to avoid conflict with my PR:) They will be removed.